XamarinCommunityToolkit/Views/DrawingView/Service/DrawingViewService.tizen.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using SkiaSharp;
+using SkiaSharp.Views.Tizen;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.Tizen;
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+	static class DrawingViewService
+	{
+		public static Stream GetImageStream(IList<Point> points,
+            Size imageSize,
+            float lineWidth,
+            Color strokeColor,
+            Color backgroundColor)
+        {
+            if (points == null || !points.Any())
+            {
+                return Stream.Null;
+            }
+
+            var image = MaxResizeImage(GetImageInternal(points.ToList(), lineWidth, strokeColor, backgroundColor),
+                                       (float)imageSize.Width, (float)imageSize.Height);
+
+            using (image)
+            {
+                var stream = image.Encode(SKEncodedImageFormat.Jpeg, 100).AsStream();
+                stream.Position = 0;
+                return stream;
+            }
+        }
+
+        static SKImage GetImageInternal(IList<Point> points,
+            float lineWidth,
+            Color strokeColor,
+            Color backgroundColor)
+        {
+            var minPointX = points.Min(p => p.X);
+            var minPointY = points.Min(p => p.Y);
+            var drawingWidth = points.Max(p => p.X) - minPointX;
+            var drawingHeight = points.Max(p => p.Y) - minPointY;
+            var bm = new SKBitmap((int)drawingWidth, (int)drawingHeight);
+            using (var gr = new SKCanvas(bm))
+            {
+                var drawingBackgroundColor = XamarinColorToDrawingColor(backgroundColor);
+                gr.Clear(drawingBackgroundColor);
+                using (var pen = new SKPaint
+                {
+                    Color = XamarinColorToDrawingColor(strokeColor),
+                    StrokeWidth = lineWidth
+                })
+                {
+                    for (var i = 0; i < points.Count() - 1; i++)
+                    {
+                        var p1 = XamarinPointToDrawingPoint(points.ElementAt(i));
+                        var p2 = XamarinPointToDrawingPoint(points.ElementAt(i + 1));
+
+                        gr.DrawLine(p1.X - (float)minPointX, p1.Y - (float)minPointY, p2.X - (float)minPointX,
+                                    p2.Y - (float)minPointY, pen);
+                    }
+                }
+            }
+
+            return SKImage.FromBitmap(bm);
+        }
+
+        static SKColor XamarinColorToDrawingColor(Color xamarinColor) => xamarinColor.ToNative().ToSKColor();
```

Comment:
Do we need to call the `ToNative()`?
```suggestion
        static SKColor XamarinColorToDrawingColor(Color xamarinColor) => xamarinColor.ToSKColor();
```

Reply:
ToSkColor doesn't exist. we need to convert to ElmSharp.Color